raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -23,6 +23,58 @@ def test_token_addresses(raiden_network, token_addresses):
     )
 
 
+@pytest.mark.parametrize("number_of_nodes", [2])
```

Comment:
Fixtures Question: I remember there must be a way to raise integration fixtures for just a single node. In this test only one node is needed, but how can that be done? Is there a fixture for that?